text/0000-view-change-method.md

```diff
@@ -35,6 +35,23 @@ The following methods are prohibited:
   * `attached_deposit`
   * `prepaid_gas`
   * `used_gas`
+  * `promise_create`
```

Comment:
Would it be better to use `promise_*` to exclude all future promise methods without having to add to the list?

Reply:
I think it's better to be explicit so that if some other promise methods is added we can remember to do the proper update.